src/main/java/net/greatstart/controllers/ProjectController.java

```diff
@@ -29,36 +30,45 @@ public ProjectController(ProjectService projectService, UserService userService)
         this.userService = userService;
     }
 
-    @RequestMapping("/project/")
+    @RequestMapping("/")
     public ModelAndView showProjects() {
         List<Project> projectList = projectService.getAllProjects();
         ModelAndView model = new ModelAndView("project/projects");
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "project/projects" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)